Donation-platform/src/main/java/com/google/model/Users.java

```diff
@@ -15,25 +15,32 @@
 package com.google.model;
 
 import java.util.Collection;
-import com.google.appengine.api.datastore.Key;
-
+import java.io.Serializable;
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import com.googlecode.objectify.annotation.Cache;
+import com.googlecode.objectify.annotation.Index;
+
+@Entity
+@Cache
 /** Represents a Users : id, userName, email, userInterests, charitiesDonatedTo. */
-public final class Users {
+public final class Users implements Serializable {
 
-  // Datastore key uniquely identifying user.
-  private Key id;
+  // Datastore String uniquely identifying user.
+  @Id private String id;
```

Comment:
Why is the ID for a charity a long and the ID for a user a string?

Reply:
The charity ID is autogenerated as a long, whereas the Users ID is provided by the Sign in OAUTH Api which is a string.

Reply:
Ok sg